src/random_seed.jl

```diff
@@ -0,0 +1,13 @@
+function get_random_seed()
+    if haskey(ENV, "SIENNA_RANDOM_SEED")
+        try
+            return parse(Int, ENV["SIENNA_RANDOM_SEED"])
+        catch e
+            val = ENV["SIENNA_RANDOM_SEED"]
+            @error("SIENNA_RANDOM_SEED $val can't be read as an integer value")
+            rethrow(e)
+        end
+        return SIENNA_RANDOM_SEED
```

Comment:
Should this be in an else clause?

Reply:
I think it is equivalent but I can put it in an else clause